CMakeLists.txt

```diff
@@ -212,6 +197,8 @@ set(EXTRA_FLAGS
     "-Werror=odr"
     "-Werror=lto-type-mismatch"
     "-Werror=strict-aliasing"
+    "-frounding-math"
```

Comment:
On FreeBSD, clang reported these weren't available. With them here, they will not be added unless the compiler supports them.